onetl/connection/db_connection/jdbc_mixin/connection.py

```diff
@@ -308,6 +308,20 @@ def execute(
             log_lines(log, str(metrics))
         return df
 
+    @property
+    def jdbc_dialect(self):
+        """
+        Returns the JDBC dialect associated with the connection URL.
+        """
+        jdbc_rdbms_name = ":".join(self.jdbc_url.split(":")[:2])
```

Comment:
Just use `self.jdbc_url`, like here:
https://github.com/MobileTeleSystems/onetl/blob/develop/onetl/connection/db_connection/jdbc_mixin/connection.py#L562-L563

Also let's use the same code for getting dialect in both places

Reply:
changed to `self.jdbc_url` however code was already in `JDBCMixin` class